Tests/scripts/update_id_set.py

```diff
@@ -305,12 +309,11 @@ def process_integration(file_path):
     """
     res = []
     if os.path.isfile(file_path):
-        if re.match(INTEGRATION_REGEX, file_path, re.IGNORECASE) or \
-                re.match(BETA_INTEGRATION_REGEX, file_path, re.IGNORECASE):
+        if checked_type(file_path, (INTEGRATION_REGEX, BETA_INTEGRATION_REGEX, PACKS_INTEGRATION_YML_REGEX)):
```

Comment:
```suggestion
        if checked_type(file_path, (INTEGRATION_REGEX, BETA_INTEGRATION_REGEX, PACKS_INTEGRATION_REGEX)):
```
if it doesn't exist, add it.

types should consist of the following:
- isfile
* regular integration
* pack integration
* beta integration
- else
* regular package integration
* pack package integration
* beta package integration